lib/assets/chart_builder/main.css

```diff
@@ -25,29 +28,39 @@
   margin-top: 12px;
 }
 
-.group {
-  display: flex;
-  flex-direction: column;
-  padding: 12px;
+.box {
+  align-self: auto;
+  flex: 0 1 25%;
   border: 1px solid var(--gray-200);
   border-radius: 5px;
-  min-height: 220px;
+  padding: 12px;
 }
 
-.group .field {
-  padding-top: 12px;
+.box:nth-child(2) {
+  display: flex;
 }
 
-.box {
+.box:nth-child(3) {
+  flex: 0 1 54%;
+}
+
+.box:nth-child(2) .field:not(:first-child) {
+  padding-left: 12px;
+}
```

Comment:
These focus specific elements, right? If so, it may be worth adding more classes to reduce mental overhead :)

Reply:
Sure thing! I’m just doing the bare minimum until we have the final design. That’s the reason why I’ve decided not to create any more classes for now.

Reply:
Ah sure, sound good to me!